src/app/components/BasicSelect.tsx

```diff
@@ -4,37 +4,27 @@ import InputLabel from "@mui/material/InputLabel";
 import MenuItem from "@mui/material/MenuItem";
 import FormControl from "@mui/material/FormControl";
 import Select from "@mui/material/Select";
-import {useQuery} from "react-query";
 
 export default function BasicSelect(props: any) {
-
-  const {handleSelectChange} = props;
-  const {location} = props;
-
-  const apiUrl =
-      "https://wi2020seb-cinema-api.azurewebsites.net/city/getAll";
-
-  const { data } = useQuery("Cities", () =>
-      fetch(apiUrl).then((res) => res.json())
-  );
+  const { handleSelectChange, cityData } = props;
+  const { location } = props;
 
   return (
-      <Box sx={{ minWidth: 120}}>
-        <FormControl fullWidth>
-          <InputLabel id="simple-select-label">Location</InputLabel>
-          <Select
-              labelId="simple-select-label"
-              id="simple-select"
-              value={location}
-              label="Location"
-              onChange={handleSelectChange}
-
-          >
-            {data?.map((location: any) => (
-                <MenuItem value={location}> {location.city}</MenuItem>
-            ))}
-          </Select>
-        </FormControl>
-      </Box>
+    <Box sx={{ minWidth: 120 }}>
```

Comment:
wollten wir nicht alles in .css speichern?

Reply:
There is an opened "Design" ticket for this sprint. When it's ready, we'll be making all design changes in one css file, so we can apply style classes to similar elements and so on. But for this sprint we've decided to leave it as it is